README.md

```diff
@@ -178,6 +175,47 @@ java -Ddks.log.directory=/var/log/dks -Ddks.log.level.console=WARN \
 -Ddks.log.level.file=INFO -jar /opt/dks/dks.jar`
 ```
 
+## Insecure mode
+
+Occasionally you may want ot bring up DKS in secure mode, for instance to
```

Comment:
@DanielChicot some typo's here. "you may want ot bring up" <-- "to"
"DKS in secure mode" <-- "DKS in insecure mode"